ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,203 @@
+import ZoneId from '../../../../../resources/zone_id.js';
+
+export default {
+  zoneId: ZoneId.MatoyasRelict,
+  timelineFile: 'matoyas_relict.txt',
+  triggers: [
+    {
+      id: 'Matoyas Mudman Hard Rock',
+      netRegex: NetRegexes.startsUsing({ id: '547F', source: 'Mudman' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '547F', source: 'Matschmann' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '547F', source: 'tadboue' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '547F', source: 'マッドマン' }),
+      condition: function(data, matches) {
+        return data.me === matches.target || data.role === 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Matoyas Mudman Petrified Peat',
+      netRegex: NetRegexes.startsUsing({ id: '5480', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5480', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5480', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5480', source: 'マッドマン', capture: false }),
+      response: Responses.moveAway('info'),
+    },
+    {
+      id: 'Matoyas Mudman Peat Pelt',
+      netRegex: NetRegexes.ability({ id: '5482', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.ability({ id: '5482', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.ability({ id: '5482', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.ability({ id: '5482', source: 'マッドマン', capture: false }),
+      alertText: (data, _, output) => output.pullOrb(),
+      outputStrings: {
+        pullOrb: {
+          en: 'Pull orb to an empty hole',
+          ja: '泥団子を四隅の穴に誘導',
+          cn: '诱导泥球到无敌人的风圈',
+        },
+      },
+    },
+    {
+      id: 'Matoyas Mudman Stone Age',
+      netRegex: NetRegexes.startsUsing({ id: '5491', source: 'Mudman', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5491', source: 'Matschmann', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5491', source: 'tadboue', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5491', source: 'マッドマン', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Matoyas Nixie Crash-smash',
+      netRegex: NetRegexes.startsUsing({ id: '598F', source: 'Nixie', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '598F', source: 'Nixchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '598F', source: 'nixe', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '598F', source: 'ノッケン', capture: false }),
+      alertText: (data, _, output) => output.awayFrom(),
+      outputStrings: {
+        awayFrom: {
+          en: 'Away from Tank and his Tether',
+          ja: 'タンクや線に離れ',
+          cn: '远离坦克及其连线',
+        },
+      },
+    },
```

Comment:
There is `Responses.awayFrom()`, which should return away from \<target>

Reply:
Yes, but also no. The tethers are also dangerous and should be warned against. Maybe `tankCleave` for the target and a special warning for others?

Reply:
Yeah, I do want to notify the tether as well, so I wrote it in paticular sentences.

Reply:
Maybe `Away from tank and tethers` is a little more succinct?
Also, that trigger runs even if you're the target of the move, which is a little odd. Maybe `condition: Conditions.TargetIsNotYou()`?

Reply:
If we're cutting the length down, maybe `Avoid tank and tethers`?